[2025-06-17 15:45:27,681] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-09-28T00:00:00+00:00 [queued]>
[2025-06-17 15:45:27,857] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-09-28T00:00:00+00:00 [queued]>
[2025-06-17 15:45:27,858] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-17 15:45:27,858] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-17 15:45:27,858] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-17 15:45:28,067] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-09-28T00:00:00+00:00
[2025-06-17 15:45:28,081] {standard_task_runner.py:52} INFO - Started process 1273 to run task
[2025-06-17 15:45:28,101] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2021-09-28T00:00:00+00:00', '--job-id', '384', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp2x2dfy0a', '--error-file', '/tmp/tmpvxtfm5k9']
[2025-06-17 15:45:28,137] {standard_task_runner.py:77} INFO - Job 384: Subtask second_task
[2025-06-17 15:45:28,702] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2021-09-28T00:00:00+00:00 [running]> on host 4f8b76136637
[2025-06-17 15:45:29,952] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-28T00:00:00+00:00
[2025-06-17 15:45:30,032] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2025-06-17 15:45:30,034] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2025-06-17 15:45:30,183] {bash.py:169} INFO - Output:
[2025-06-17 15:45:30,194] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2025-06-17 15:45:30,194] {bash.py:177} INFO - Command exited with return code 0
[2025-06-17 15:45:31,146] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210928T000000, start_date=20250617T154527, end_date=20250617T154531
[2025-06-17 15:45:32,936] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-17 15:45:33,537] {local_task_job.py:146} INFO - Task exited with return code 0
